link/src/main/java/com/stripe/android/link/serialization/PopupPayload.kt

```diff
@@ -149,7 +149,7 @@ internal data class PopupPayload(
                 paymentObject = paymentObject(),
                 intentMode = stripeIntent.toIntentMode().type,
                 setupFutureUsage = stripeIntent.isSetupForFutureUsage(),
-                flags = flags,
+                flags = flags.plus("link_2024_rebrand_m1" to LinkUi.useNewBrand),
```

Comment:
Is this passed separate of the `Link` flags provided from `LinkSettings`?

Reply:
Looks like we get it in `link_settings`, so this is redundant now!